paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentOptionFactory.kt

```diff
@@ -48,6 +50,12 @@ internal class PaymentOptionFactory @Inject constructor(
             loadIcon(darkThemeIconUrl)
         } else if (lightThemeIconUrl != null) {
             loadIcon(lightThemeIconUrl)
+        } else if (@Suppress("DEPRECATION") paymentOption.drawableResourceId == MISSING_DRAWABLE_RESOURCE) {
+            errorReporter.report(
+                ErrorReporter.UnexpectedErrorEvent.PAYMENT_OPTION_MISSING_ICON_URL_AND_RES,
+                additionalNonPiiParams = mapOf("payment_option" to paymentOption.label)
```

Comment:
I'm not sure this is super useful. Might be worth removing.

Reply:
Ah fair, but this is a genuine "unexpected" error and I think I'd like to know if it happened, so I'm going to leave it

Reply:
updated per slack